grype/presenter/template/presenter.go

```diff
@@ -84,5 +85,19 @@ var funcMap = func() template.FuncMap {
 
 		return 0
 	}
+	f["byName"] = func(collection interface{}) interface{} {
```

Comment:
since this function name can be used anywhere in the template, and the function is only intended for matches, we should probably make the name more explicit to convey proper usage.

Something like `byMatchName` or `sortByMatchName`, so there is a hint that if you aren't using this on matches that you're probably using it incorrectly.

Reply:
`byMatchName` I think reads correctly here based on other templating languages. I'll update it to that.